styles.css

```diff
@@ -7,3 +7,155 @@ body {
   background: #f7f7f9;
   overflow-x: hidden;
 }
+
+/* Toolbar */
+
+.container {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+.toolbar {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 114px;
+  height: 48px;
+  left: 12px;
+  top: calc(50% - 48px);
+}
+
+.welcome {
+  width: 90px;
+  height: 28px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #28352f;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.menu-button {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 1px;
+  top: calc(50% - 40px);
+  border-radius: 8px;
+}
+
+.template-icon {
+  width: 24px;
+  height: 35px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+/* Headline */
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 325px;
+  height: 726px;
+  left: 0;
+  top: 92px;
+}
+
+.primary {
+  width: 327px;
+  height: 192px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
```

Comment:
- [ ] These lines are unnecessary as this container don't have `display: flex` and these properties work if the container has a `display: flex;`
I would suggest removing that for the sake of DRY principle.